README.md

```diff
@@ -84,74 +127,16 @@ sqlite3 app.db
 
 Our scraper crawls Yale's websites in order to obtain the data we provide. See documentation [here](app/scraper/README.md).
 
-## Making changes
-
-In order to contribute your changes to our codebase, there are a few steps you must go through. Please reach out to your development team lead with any questions.
-
-### Definitions:
-
-- **upstream**: the "official” version of the repository (https://github.com/Yalies/api)
-- **fork**: a copy of the upstream repository that you create on your own user account and have full access to
-- **remote**: git’s local record of where on the internet the repository is hosted
-- **origin**: the default remote that is generated when you clone a repository
-- **branch**: a separate copy of the code within the same repository
-- **master**/**main**: the default branch in git repositories
-
-### Submitting changes
-
-Pull the latest code from the upstream repository to ensure you're working with the newest:
-
-```sh
-git pull upstream master
-```
-
-Switch to a new branch to hold your changes:
-
-```sh
-git checkout -b changes_description
-```
-
-The name of the branch should be short and refer to what you’re planning to change.
-
-Next, make your code changes! Be sure to test them and make sure the app runs as you expect.
-Next, commit your code:
+## Developer practices
 
-```sh
-# To tell git to track all the files you changed
-git add -A
-# To label this set of changes:
-git commit -m "Describe your changes here"
-# Make sure to title your commit in the imperative tense, for example "Add new features” instead of "Added…", "Adding…", etc.
-```
-
-Next, upload your code to your fork:
-
-```sh
-git push -u origin your_branch_name
-```
-
-**NOTE:** If you make another change on this branch, you can just do `git push` (without additional flags) and it will automatically push to the last remote/branch you specified.
-
-Next, create a pull request (a request to merge your changes into the main repository) by going to the repository page on GitHub and clicking the green "Compare & Pull Request” button that appears.
-
-Title the pull request with a description of all included changes.
-
-In the description, write "Fixes #X” or "Resolve #X”, with X being an issue number, for each issue you're fixing in this PR. This will save time by telling GitHub to automatically close those issues once your changes are merged.
-
-On the right side under Reviewers, select everyone else on our team, particularly your team lead.
-
-Congratulations! Your changes will be up for review. After they are merged, you'll need to check out back to master and pull your changes into master from the upstream repository.
-
-```sh
-git checkout master
-git pull upstream master
-```
+To ensure the quality of our codebase, please follow these best practices. Refer to the [Git tutorial video](https://www.youtube.com/watch?v=yZo-aF1dqhs) for help.
```

Comment:
Aw. I like your video, but I don't want to remove all the textual guidance too.

Reply:
Having now watched the video, it does not seem to cover the entirety of the workflow. I would like it if this guide could stick around, although you're welcome to promote your video as aggressively as you want to help people get general intuition about git.

Reply:
OK, I'll add it back in